src/test/java/com/epam/tc/hw1/BaseTest.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.tc.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+public class BaseTest {
+
+    public Calculator calculator = new Calculator();
+
+    @BeforeTest
+    public void before() {
+        calculator = new Calculator();
+    }
+
+    @AfterTest
```

Comment:
It will be better to use `@AfterMethod` instead of `@AfterTest`.
`@ AfterTest` does not work before each method which marked `@Test` annotation